tests/trnsysGUI/diagram/testEditor.py

```diff
@@ -136,6 +156,25 @@ def testMassFlowSolver(self, testProject: _Project, qtbot, monkeypatch) -> None:
         helper = _Helper(testProject)
         helper.setup()
 
+        mainWindow = self._createMainWindow(helper, qtbot, monkeypatch)
+
+        self._exportMassFlowSolverDeckAndRunTrnsys(mainWindow.editor)
+
+        massFlowSolverDeckFileName = f"{testProject.projectName}_mfs.dck"
+        helper.ensureFilesAreEqual(massFlowSolverDeckFileName)
+
+        massFlowRatesPrintFileName = f"{testProject.projectName}_Mfr.prt"
+        helper.ensureCSVsAreEqual(massFlowRatesPrintFileName)
```

Comment:
@zuckerruebe:
Pandas provides dataframe comparison functions for testing:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html